### PR TITLE
🖼️ Complete the proposals current page layout

### PR DESCRIPTION
**Pull request by thesan**
See #541.

### CODE REVIEW

File: packages/ui/src/common/components/List/ListHeader.tsx [hunk 1]
```diff
@@ -6,7 +6,6 @@ import { TableListRowStyle } from './List'
 
 export const ListHeaders = styled.div`
   display: grid;
-  grid-area: accountstablenav;
```
Comment (jodator) [inline packages/ui/src/common/components/List/ListHeader.tsx:9]: :+1:

File: packages/ui/src/proposals/hooks/useProposals.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+import { useMockDelay } from '@/mocks/hooks/useMockDelay'
+
+import { Proposal } from '../types'
+
+const mock: { [k: string]: { proposals: Proposal[] } } = {
+  current: { proposals: mockProposals },
+  past: { proposals: [] },
+}
+
+export const useProposals = ({ type }: { type: 'current' | 'past' }) => {
```
Comment (jodator) [inline packages/ui/src/proposals/hooks/useProposals.ts:11]: A bit cleaner approach would be to move that to an interface.

Comment (jodator) [inline packages/ui/src/proposals/hooks/useProposals.ts:11]:
```suggestion
const mock: { [k: string]: { proposals: Proposal[] } } = {
  current: { proposals: mockProposals },
  past: { proposals: [] },
}

interface Props {
  type: 'current' | 'past'
}

export const useProposals = ({ type }: Props) => {
```

Like this: :point\_up: Then you can extend the interface if needed and the function is easier to read.

**Review by jodator — CHANGES_REQUESTED** [review]
Thanks! One question though, I'd move the mentioned types definition to an interface so the hook function is easier to read.

**Comment by jodator** [comment]
@thesan ps.: please remember to add `See #xxx` or `Closes #xxx` in the PR description to have a direct link to an issue.

If you see that this PR resolves all the requirements of the issue, please use `Closes #xxx`.

**Comment by thesan** [comment]
> @thesan ps.: please remember to add `See #xxx` or `Closes #xxx` in the PR description to have a direct link to an issue.
> 
> If you see that this PR resolves all the requirements of the issue, please use `Closes #xxx`.

Got it, I will keep it in mind :ok_hand:

**Pull request merged**